### DISCUSSION TITLE
Original Build

### DISCUSSION THREAD

**Discussion by martinwoodward** [martinwoodward/octolamp#1] [Show and tell]
I thought I'd start with some pictures from my original build - connected to my Home Assistant instance.

![IMG_9122](https://user-images.githubusercontent.com/856858/212750774-7a008355-303f-41c5-8a35-f2f5c8ca8516.jpg)

![IMG_9132 2023-01-16 16_26_21](https://user-images.githubusercontent.com/856858/212750794-4e025c06-cf66-478c-a910-513a45ed5dcb.gif)

![IMG_9128](https://user-images.githubusercontent.com/856858/212750910-dad1c773-2904-40d2-8941-b50ed15e445b.jpg)

At the moment these are my coolest presets:

<details>
<summary>Pink Ripples</summary>

```
{"on":true,"bri":128,"transition":7,"mainseg":0,"seg":[{"id":0,"start":0,"stop":102,"grp":1,"spc":0,"of":0,"on":true,"frz":false,"bri":255,"cct":127,"col":[[255,200,0],[102,255,212],[0,0,0]],"fx":101,"sx":128,"ix":128,"pal":13,"c1":8,"c2":20,"c3":16,"sel":true,"rev":false,"mi":false,"o1":false,"o2":false,"o3":false,"si":0,"m12":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0}]}
```

</details>
<details>
<summary>Polarized Rainbow</summary>

```
{"on":true,"bri":128,"transition":7,"mainseg":0,"seg":[{"id":0,"start":0,"stop":102,"grp":1,"spc":0,"of":0,"on":true,"frz":false,"bri":255,"cct":127,"col":[[255,160,0],[0,0,0],[0,0,0]],"fx":110,"sx":128,"ix":128,"pal":54,"c1":128,"c2":128,"c3":16,"sel":true,"rev":false,"mi":false,"o1":false,"o2":false,"o3":false,"si":0,"m12":1},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0}]}
```

</details>

**Comment by wesmacdonald** [comment]
Definitely want one of these, very nice!

**Reply by skyj43v3l** [reply]
Awesome! Would love one but I don't have access to a 3D Printer. What's the solution? Anyone selling one? 🤓

**Comment by skyj43v3l** [comment]
Awesome! Would love one but I don't have access to a 3D Printer. What's the solution? Anyone selling one? 🤓

**Reply by flow23** [reply]
There are a lot of 3d printing services out there.
